src/components/Article.jsx

```diff
@@ -1 +1,32 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => {
```

Comment:
better destructure props inside of components, this will make future expanding easier